src/pages/iv-items/forms/ItemProductionForm.js

```diff
@@ -101,6 +101,8 @@ export default function ItemProductionForm({ labels, editMode, maxAccess, store
                   if (newValue) {
```

Comment:
By writing this code below in onchange, you may simplify it. 
   formik.setFieldValue('lineId', newValue?.recordId || '')       
   formik.setFieldValue('lineId', '')